website/server/libs/cron.js

```diff
@@ -130,6 +130,14 @@ function trackCronAnalytics (analytics, user, _progress, options) {
 
 function awardLoginIncentives (user) {
   if (user.loginIncentives > 50) return;
```

Comment:
Something for the future, but this should be a constant instrwad of a magic number.